k8s-daemonset/README.md

```diff
@@ -352,3 +352,11 @@ buoyantio/helloworld docker image:
 ```bash
 docker run --rm --entrypoint=helloworld-client buoyantio/helloworld:0.1.2 $L5D_INGRESS_LB:4140
 ```
+
+## kubectl Docker container
+
+These examples make use of a
+[kubectl Docker container](https://hub.docker.com/r/buoyantio/kubectl/), hosted
+in the [buoyantio DockerHub repo](https://hub.docker.com/r/buoyantio/). To build
+this container yourself, follow the instructions in the
+[Kubernetes Repo](https://github.com/kubernetes/kubernetes/tree/master/examples/kubectl-container).
```

Comment:
Thanks for adding this. I was actually thinking it would make sense in [docker/README.md](https://github.com/linkerd/linkerd-examples/blob/master/docker/README.md), rather than in this file. Mind moving it there? Am hoping the majority of users of this directory won't need to rebuild the image.